01_temperature/temperature.rb

```diff
@@ -0,0 +1,20 @@
+def ftoc(val)
+  val = val-32
+  5.0*val/9.0
+
+=begin
+  if val==32
+    0
+  else
+    5.0*(val-32)/9.0
+  end
+=end
```

Comment:
Learn about good comments: https://blog.submain.com/field-guide-code-comments/

Reply:
Got it!